profiles/package.mask

```diff
@@ -268,6 +268,7 @@ dev-libs/libFuzzer
 # Mask followup bugfixes for =sys-apps/attr-2.4.48 as well until proper
 # testing has been conducted.
 =sys-apps/acl-2.2.52-r2
+>=app-forensics/aide-0.16-r1
```

Comment:
Is this really necessary? I mean, did you test the patched package with `<sys-apps/attr-2.4.48`? I only masked `=sys-apps/acl-2.2.52-r2` because it's a system-critical package that can do big damage if it breaks.

Reply:
Yes. aide-0.16-r1 won't build with attr < 2.4.48, because ENOATTR definition was moved from attr/xattr.h to attr/attributes.h, so I cannot just drop attr/xattr.h include. Thus aide-0.16-r1 and attr-2.4.48 must be unmasked simultaneously.

Reply:
You can simply define it by yourself. Other projects have done it the same: 

https://github.com/fdupoux/fsarchiver/pull/54/commits/57759f2b1106b6d19f3bc11b21463f4307dfc3d8

That way you can retain backwards compatibility.

Reply:
Ok, will update PR sometime during the week then.

Reply:
All done.